src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleEqualityTests.cs

```diff
@@ -2191,6 +2191,7 @@ public static void Main()
 Operator '!=' cannot be applied to operands of type 'System.ValueTuple<int,int,int>' and 'System.ValueTuple<int,int,int>'");
         }
 
+        // PROTOPTYPE(nullable-api): Support deconstruction assignment
```

Comment:
@333fred It looks like some PROTOTYPE markers made it into `master`. Please scrub and replace with github issues.